src/main/java/net/sf/jabref/logic/layout/LayoutEntry.java

```diff
@@ -644,9 +646,14 @@ private LayoutFormatter getLayoutFormatterByName(String name) throws Exception {
                             int startParam = i;
                             i++;
                             boolean escaped = false;
-                            while (((i + 1) < c.length) && !(!escaped && (c[i] == '"') && (c[i + 1] == ')'))) {
+                            while (((i + 1) < c.length)
```

Comment:
I vaguely remember that I have already seen some code which took care of balanced braces. Probably in the BibtexParser. Could you please check there and maybe extract some common code (to StringUtil?).